apps/modernization-ui/src/apps/search/event/components/InvestigationSearch/InvestigationGeneralFields.tsx

```diff
@@ -15,13 +14,13 @@ import { SearchCriteriaContext } from 'providers/SearchCriteriaContext';
 import { ChangeEvent, ReactElement } from 'react';
 import { Controller, UseFormReturn, useWatch } from 'react-hook-form';
 import { formatInterfaceString } from 'utils/util';
+import { UserAutocomplete } from 'options/autocompete/UserAutocomplete';
 
```

Comment:
Name should be changed so it doesn't clash potential class or confuse with `public static userAutocomplete`